devops/demo/landing-page/webroot/index.html

```diff
@@ -5,6 +5,14 @@
         <link rel="stylesheet" href="/assets/css/demo.css">
     </head>
     <body>
+        <!-- Apache-licensed "content copy" icon from https://material.io/resources/icons/ -->
+        <svg style="display:none">
```

Comment:
I'm a bit surprised this page doesn't already have a CSP, but let's avoid using an inline style; instead we can have a "hide" CSS class that just applies display none.

Reply:
Done.